src/StripeTests/Entities/Events/EventTest.cs

```diff
@@ -19,7 +19,7 @@ public void Deserialize()
 
             Assert.NotNull(evt.Data);
             Assert.NotNull(evt.Data.Object);
-            Assert.IsType<Customer>(evt.Data.Object);
+            Assert.IsType<Plan>(evt.Data.Object);
```

Comment:
Maybe we should just remove this assertion and add tests with local fixtures to ensure that resources in events are correctly deserialized.

Reply:
fixed

Reply:
Thank You👌